web/src/components/filings-calendar/tax-access-modal/TaxAccessStepTwo.tsx

```diff
@@ -44,7 +51,7 @@ export const TaxAccessStepTwo = (props: Props): ReactElement => {
   const { queueUpdateTaskProgress } = useUpdateTaskProgress();
   const business = props.CMS_ONLY_fakeBusiness ?? updateQueue?.currentBusiness();
 
-  const [profileData, setProfileData] = useState<ProfileData>(createEmptyProfileData());
+  const [tempProfileData, setTempProfileData] = useState<ProfileData>(createEmptyProfileData());
```

Comment:
[sand] I know this was discussed in a previous comment, but I'm not sure that I agree with this name change. Fully recognize that the naming might be confusing, but this is a pattern we follow throughout the application. If we want to change how this is handled, I'd rather us address that in a single refactor and keeping naming consistent.

Same comment for `SidebarCardFundingNudge`.